aptos-move/framework/aptos-framework/sources/governance_proposal.move

```diff
@@ -22,8 +25,8 @@ module aptos_framework::governance_proposal {
         metadata_location: String,
         metadata_hash: String,
     ): GovernanceProposal {
-        assert!(length(&metadata_location) <= 256, error::invalid_argument(ETOO_LONG));
-        assert!(length(&metadata_hash) <= 256, error::invalid_argument(ETOO_LONG));
+        assert!(length(&metadata_location) <= 256, error::invalid_argument(EMETADATA_LOCATION_TOO_LONG));
+        assert!(length(&metadata_hash) <= 256, error::invalid_argument(EMETADATA_HASH_TOO_LONG));
```

Comment:
This change is causing an error in `test_metadata_hash_too_long`: the code used to be `ETOO_LONG = 1` and now it's `EMETADATA_HASH_TOO_LONG = 2`:

```
┌── test_metadata_hash_too_long ──────
│ error[E11001]: test failure
│    ┌─ /Users/alinush/repos/aptos-core/aptos-move/framework/aptos-framework/sources/governance_proposal.move:29:9
│    │
│ 24 │     public(friend) fun create_proposal(
│    │                        --------------- In this function in 0x1::governance_proposal
│    ·
│ 29 │         assert!(length(&metadata_hash) <= 256, error::invalid_argument(EMETADATA_HASH_TOO_LONG));
│    │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Test did not abort with expected code. Expected test to abort with 65537 but instead it aborted with 65538 here
```